core/test-runtime/src/system.rs

```diff
@@ -248,6 +248,12 @@ mod tests {
 	use ::{Header, Digest, Extrinsic, Transfer};
 	use primitives::{Blake2Hasher};
 	use primitives::storage::well_known_keys;
+	use substrate_executor::WasmExecutor;
+
+	lazy_static! {
```

Comment:
does this really need to be `lazy_static`? `include_bytes` evaluates to a `&'static [u8]`, so wouldn't a plain old static be fine?

Reply:
True; not needed in this case. Removed lazy_static.